README.md

```diff
@@ -2,11 +2,18 @@
 
 Before starting the installation process, you need:
 
-- An Ubuntu 20.04/22.04 machine with SSH credentials
-  - This must have ports 22 (SSH), 80 (HTTP), 443 (HTTPS) and 6443 (Kubernetes) exposed
+- Target building resources.
+  - Ubuntu 20.04/22.04 machine(s) with SSH credentials. 
+    - At least one, but also the script can also work for multiple nodes. The hostname of each node can be called `node0`, `node1`, etc.
+    - `node0` must have ports 22 (SSH), 80 (HTTP), 443 (HTTPS) and 6443 (Kubernetes) exposed. `node0` must have a static public IP address that allows domain name to point to. `node<x>` must have post 22 (SSH) exposed. All nodes are better to be in the same vlan so they can communicate with each other.
```

Comment:
I'm not sure this is entirely correct. In my setup, there is a load balancer on all nodes HTTP and HTTPS. Also, if you use the `HA_CLUSTER`, then you can connect to the Kubernetes API on subsequent nodes.

I think for simplicity, just saying "expose these four ports" is ok.

Reply:
Yes, I agree it should be more general than to just talking about my own setting. I have applied the changes and also updated this part a bit.